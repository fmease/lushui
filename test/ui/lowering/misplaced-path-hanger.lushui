use foo.bar.(super.ban, topmost, self)
