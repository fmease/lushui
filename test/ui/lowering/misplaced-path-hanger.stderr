error[E026]: path ‘super’ not allowed in this position
  ┌─ ${TEST_FOLDER}/lowering/misplaced-path-hanger.lushui:1:14
  │
1 │ use foo.bar.(super.ban, topmost, self)
  │              ═════
  │
 help: consider moving this path to a separate use-declaration

error[E026]: path ‘topmost’ not allowed in this position
  ┌─ ${TEST_FOLDER}/lowering/misplaced-path-hanger.lushui:1:25
  │
1 │ use foo.bar.(super.ban, topmost, self)
  │                         ═══════
  │
 help: consider moving this path to a separate use-declaration

error: aborting due to 2 previous errors

